site/components/layout/__snapshots__/CookieBanner.test.tsx.snap

```diff
@@ -0,0 +1,3 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
+exports[`CookieBanner > should render correctly 1`] = `null`;
```

Comment:
Because hideBanner is instantiated as true, this snapshot will always show the component as 'null'.

If we're going to do snapshots on this component, we probably need to do 3, for the 3 different ways it can be rendered.

Reply:
done